features/__snapshots__/task_validate_image.snap

```diff
@@ -272,6 +272,14 @@ policy:
     - github.com/enterprise-contract/ec-policies//policy/lib
 success: true
 
+[Golden container image:show-config - 1]
+{
+  "include": [
+    "slsa_provenance_available"
+  ]
+}
+---
+
```

Comment:
What I expected to see here is the full content of the config, i.e. the whole thing from line 13 to line 28 in [features/task_validate_image.feature](https://github.com/enterprise-contract/ec-cli/pull/2460/files#diff-658ed1262bea09265fe40c9bf025b6b9d126a20e5800c4db958767b7c791cbbb)

This looks like just a part of it.

Not sure why, but see if you can figure it out maybe..?

Reply:
Do we expect all the scenarios in [features/task_validate_image.feature](https://github.com/enterprise-contract/ec-cli/pull/2460/files#diff-658ed1262bea09265fe40c9bf025b6b9d126a20e5800c4db958767b7c791cbbb) expect a check for the show-config? I only add [Golden container image:show-config - 1] right now. Thanks.